### ISSUE TITLE
Is websocket <bitcoinAddress> room still working?

### ISSUE THREAD

**Issue by deoxen0n2** [bitpay/insight-api#485]
Hi, I tried subscribing to `<bitcoinAddress>` room and listen on `<bitcoinAddress>` event but nothing happens when I create transaction to this address. The code I used is the following:

```js
socket.on('connect', () => socket.emit('subscribe', 'my_address_here'))
socket.on('my_address_here', data => console.log('new address data:', data))
```
But this code works (copied from `insight-ui`):

```js
socket.on('bitcoind/addresstxid', data => console.log('new address data:', data))
socket.on('connect', () => socket.emit('subscribe', 'bitcoind/addresstxid', [ 'my_address_here' ]))
```

I tested on https://test-insight.bitpay.com. Sorry if I did something wrong.

**Comment by braydonf** [comment]
Yep the latter will work. Using the address as the name of the event doesn't work well with the bus subscription model of [bitcore-node](https://github.com/bitpay/bitcore-node). We should probably make a note of this.

**Comment by aravind-ig** [comment]
`socket.on('bitcoind/addresstxid', data => console.log('new address data:', data))
socket.on('connect', () => socket.emit('subscribe', 'bitcoind/addresstxid', [ 'my_address_here' ]))`
This code above works for [test-insight.bitpay.com](https://test-insight.bitpay.com) but not for the Insight Node I set up. The socket connects and all other events are received well except for the **<bitcoinAddress> room**.
My Insight Node : 
Version | **110200**
Protocol version | **70002**

**Comment by roccomuso** [comment]
Any news on this? having same issue on `https://insight.bitpay.com`
